tests/integration/cht-form/wdio.conf.js

```diff
@@ -1,4 +1,4 @@
-const wdioBaseConfig = require('../wdio.conf');
+const wdioBaseConfig = require('../../e2e/wdio.conf');
```

Comment:
Minor, but what do you think about moving this `wdio.conf` file up a level so it is just `tests/wdio.conf` (and not nested in `tests/e2e`)?  I just think that might make it a bit easier to remember that it is getting referenced from outside the `tests/e2e` folder....

If it looks like it might be tricky to move `wdio.conf`, then def don't worry about it!